js-OOP/classes1.js

```diff
@@ -0,0 +1,53 @@
+// Task 4
+class CarClass {
+    constructor(make, model, year) {
+      this.make = make;
+      this.model = model;
+      this.year = year;
+      this.speed = 0;
+    }
+  
+    getCarInfo() {
+      return `Car Info: ${this.year} ${this.make} ${this.model}`;
+    }
+  
+    accelerate(speed) {
+      this.speed += speed;
+    }
+  
+    brake(speed) {
+      this.speed -= speed;
+    }
+  }
+  
+  class AudiClass extends CarClass {
+    constructor(make, model, year, color) {
+      super(make, model, year);
+      this.color = color;
```

Comment:
color must be private
```suggestion
      this.#color = color;
```